static/styles/modal.css

```diff
@@ -205,3 +205,16 @@
         fill: hsl(0, 0%, 100%);
     }
 }
+
+#copy_email_address_modal {
+    width: 800px;
+
+    .stream-email {
+        padding: 10px;
+        font-size: 0.85rem;
+    }
```

Comment:
We already have `padding` and `font-size` properties defined for `stream-email` class in `subscriptions.css`. We can modify it there itself.